src/app/docs/button/button-docs.component.html

```diff
@@ -95,6 +95,29 @@
               </p>
             </td>
           </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              <button
+                type="button"
+                class="sky-btn sky-btn-icon-borderless"
+              >
+              <i [skyThemeClass]="{
```

Comment:
I tried to serve locally so that I can see the updates while the PR checks aren't passing, and I get an error that we can't bind `skyThemeClass` since isn't a known property of `i`.